example/linux/installation.md

```
@@ -0,0 +1,178 @@
+### Additional Steps for Troubleshooting
+
+1. Verify Node.js and `tsx` installation:
+   ```bash
+   node -v
+   tsx -v
+   ```
+2. Ensure MongoDB and Redis are running:
+   ```bash
+   sudo systemctl status mongod
+   sudo systemctl status redis
+   ```
```

Markdownlint (0.37.0)

Files should end with a single newline character

(MD047, single-trailing-newline)